ansible/tasks/vault_kubernetes_auth_method_config.yml

```diff
@@ -23,6 +23,8 @@
 - name: Get vault token
   set_fact:
     vault_token: "{{ vault_login.stdout | from_json | community.hashi_vault.vault_login_token }}"
+  vars:
```

Comment:
VENV containing vault module not needed to execute this ansible command.
We should remove this var from the execution of this task